alphabase/io/tempmmap.py

```diff
@@ -51,6 +51,44 @@ def _check_temp_dir_existence(abs_path: str) -> str:
             f"The directory {abs_path} in which the file should be created does not exist."
         )
 
+def _check_file_location(abs_file_path: str, overwrite = False) -> str:
```

Comment:
similar to your feedback in the review I also took the opportunity to move the other longer if else block to a separate function for better readability. @mschwoer let me know what you think.

Reply:
+1, good idea!
just a small nitpicking on the name: from a method that is named `check_..` I would not expect a return value .. mb rename to `_get_file_location` ?

Reply:
That's true. I chose check because we already provide the file location before it's just a sanity check to make sure everything is as expected before proceeding. Happy to change though if you think its better understandable!

Reply:
renamed to `_get_file_location`.